.github/workflows/demo.yml

```diff
@@ -0,0 +1,60 @@
+name: test notebooks
+
+# this aims to test the notebooks, the way the user would
+# - install package from github
+# - use the nbmake pytest plugin to run the notebooks from top to bottom
+
+#
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+
+  test_notebooks:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-10.15, windows-latest]
+        python-version: ["3.9"]
+        notebook: [usage_pretrained_weights]
+        include:
+          - os: ubuntu-latest
+            path: ~/.cache/pip
+          - os: macOS-10.15
+            path: ~/Library/Caches/pip
+          - os: windows-latest
+            path: ~\AppData\Local\pip\Cache
+
+    steps:
+      - name: Clone repo
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install deepmreye from github
+        run: pip install git+https://github.com/Remi-Gau/DeepMReye@setup
```

Comment:
Note that this way of installing tries to mimic how a user would install the demo.

Because the masks are not downloaded if installed from the upstream repo (see #6 and this [comment PR10](https://github.com/DeepMReye/DeepMReye/pull/10#issuecomment-1176266742)), I am currently installing from a fork on my branch where this problem has been fixed.